src/main/java/de/tum/in/www1/artemis/config/CacheConfiguration.java

```diff
@@ -86,6 +86,7 @@ public CacheManager cacheManager(HazelcastInstance hazelcastInstance) {
 
     /**
      * Setup the hazelcast instance based on the given jHipster properties and the enabled spring profiles.
+     * 
```

Comment:
```suggestion
     *
```
A trailing whitespace is automatically added to the empty lines. Can this be removed somehow?

Reply:
Great catch. This seems to be an unresolved issue in the eclipse formatter, see https://stackoverflow.com/questions/15755671/how-to-stop-eclipse-formatter-from-adding-trailing-whitespaces-in-javadoc. Here the only solution is to run this as an additional taks, but nothing integrated into the formatter. I also tried updating the formatter version, but this is blocked due to https://github.com/diffplug/spotless/issues/1046

So I told IntellIJ to reformat everything, which removed that spaces. This isn't the best solution, but the only one I could come up with.